### PR TITLE
Pass notification title to android module

### PR DESCRIPTION
**Pull request by LMakshow**
Thanks for the amazing library and maintenance!
I want to add a prop to pass the notification title to Download Manager. On my Android device (Xiaomi 11 Ultra), if `isNotificationVisible === true`, when the download starts, the notification from the Download Manager is shown as "untitled" and then the title changes to a random number like "2346436.".
With the new `notificationTitle` prop, we can pass the title of the file explicitly to the Download Manager to show it both in the notification drawer and the download list.

telegram-cloud-photo-size-2-5415886020488062830-y

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -148,6 +149,7 @@ export function download (options: DownloadOptions) {
 
   RNBackgroundDownloader.download({
     ...options,
+    notificationTitle: options.notificationTitle,
```
Comment (seniv) [inline index.ts]: It seems like we can omit this line as we already spread `options` object

**Review by seniv — APPROVED** [review]
LGTM! only one minor comment

**Comment by LMakshow** [comment]
@seniv You are correct!

**Comment by kesha-antonov** [comment]
Thanks! 🤝

**Pull request merged**